registry/example/aggregates.go

```diff
@@ -0,0 +1,49 @@
+package example
+
+import (
+	"context"
+
+	eh "github.com/looplab/eventhorizon"
+	"github.com/looplab/eventhorizon/registry"
+)
+
+// HereAggregate example aggregate
+type HereAggregate struct {
+	registry.RaiseEvent
```

Comment:
It would be nice to name this reflection.AggregateBase instead, to conform to the same naming as the simpler base.